scalafix/rules/src/main/scala/rsc/rules/RscCompat.scala

```diff
@@ -30,9 +31,20 @@ case class RscCompat(legacyIndex: SemanticdbIndex, config: RscCompatConfig)
 
   override def fix(ctx: RuleCtx): Patch = {
     val targets = collectRewriteTargets(ctx)
-    targets.map(ascribeInferredType(ctx, _)).asPatch
+
+    val typeAscriptions = targets.map(ascribeInferredType(ctx, _)).asPatch
+
+    val addedImports = if (config.better) {
+      new GlobalImports(ctx).addGlobalImports(addedImportsScope.importers)
+    } else {
+      Patch.empty
+    }
+
+    typeAscriptions + addedImports
   }
 
+  private val addedImportsScope: AdditionalImportsScope = new AdditionalImportsScope
```

Comment:
How about calling this `AddedImportsScope` to be consistent?

Reply:
renamed